webmention.php

```diff
@@ -81,13 +114,34 @@ public static function parse_query($wp) {
     if (!array_key_exists('webmention', $wp->query_vars)) {
       return;
     }
-
-    $content = file_get_contents('php://input');
-    parse_str($content);
+    else {
+      // check if the end point has expired
+      $valid_ticks = array( 0, -1, -2 );
```

Comment:
Why are numbers negative here? And why have an array of numbers combined with a `foreach` rather than just a regular `$length = 3; for ($i = 0; $i < $length; $i++) {}`?